nixos/modules/services/monitoring/prometheus/exporters/folder-size.nix

```
@@ -0,0 +1,84 @@
+{ config, lib, pkgs, options, utils }:
+let
+  cfg = config.services.prometheus.exporters.folder-size;
+
+  settingsFormat = pkgs.formats.json { };
+  settingsFile = settingsFormat.generate "config.json" (builtins.filter (folder: builtins.hasAttr "path" folder) cfg.folders);
```

`filter` here does nothing and can be dropped afaiu
```suggestion
  settingsFile = settingsFormat.generate "config.json" cfg.folders;
```

---

I think I may have left it in because of local testing/useage